### PR TITLE
Updated docs for wallet contracts

### PR DESCRIPTION
**Pull request by ScreamingHawk**
* Added content to empty pages
* Moved generic content out of v1 folder
* Added missing v2 content

### CODE REVIEW

File: docs/pages/solutions/technical-references/internals/v1/guest-module.mdx [hunk 1]
```diff
@@ -1,2 +1,13 @@
-# GuestModule and on-demand deployment
+# Guest Module
 
+The Guest Module is an implementation of the Sequence wallet that does not signatures, nonces or replay protection.
```
Comment (moskalyk) [inline docs/pages/solutions/technical-references/internals/v1/guest-module.mdx]: should this read 'does no' signatures, nonces, or replay protection

File: docs/pages/solutions/technical-references/internals/v1/guest-module.mdx [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+Executing transactions using this wallet is not an authenticated process, and can be done by any address.
+
+This contract is completely public with no security, designed to execute pre-signed transactions and use Sequence tools without using the wallets.
+
+## On Demand deployment
+
+When a user interacts with their wallet on chain for the first time, the transaction will include the deployment of the wallet contract.
```
Comment (moskalyk) [inline docs/pages/solutions/technical-references/internals/v1/guest-module.mdx]: across documentation and website we use the term 'on-chain' with a hyphen

Comment (ScreamingHawk) [reply docs/pages/solutions/technical-references/internals/v1/guest-module.mdx]: Found a few other instances of "on chain". Have updated in this PR as well

File: docs/pages/solutions/technical-references/internals/v1/guest-module.mdx (hunk 1 above)
Comment (ScreamingHawk) [reply docs/pages/solutions/technical-references/internals/v1/guest-module.mdx]: Using "does not support"

**Review by moskalyk — APPROVED** [review]

**Comment by moskalyk** [comment]
looks good to me

**Pull request merged**